spec/player_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'players'
+
+describe Player do
+  let(:player) { Player.new('Uzair', 'Saad') }
+  context 'It Initializes the players' do
+    it 'Checks if the passed values are correct' do
+      player1 = player.player1
+      player2 = player.player2
+      expect(player1).to eq('Uzair')
+      expect(player2).to eq('Saad')
+    end
+  end
+end
```

Comment:
- [ ] This is a typical example of a negative test case for this test.
`it 'Checks if the passed values are incorrect' do
      player1 = player.player1
      player2 = player.player2
      expect(player1).not_to eq('George')
      expect(player2).not_to eq('Hommer')
    end`